app/services/usps_in_person_proofing/enrollment_helper.rb

```diff
@@ -1,10 +1,10 @@
 module UspsInPersonProofing
   class EnrollmentHelper
-    def save_in_person_enrollment(user, profile, pii)
+    def save_in_person_enrollment(user, profile, pii, selected_location_details = nil)
       enrollment = InPersonEnrollment.create!(
         profile: profile,
         user: user,
-        current_address_matches_id: pii[:same_address_as_id],
+        current_address_matches_id: pii['same_address_as_id'],
```

Comment:
This might have been working incorrectly before as well. Note below in `create_usps_enrollment` we pick out `pii` attributes from the hash as string keys, not symbols. I confirmed in the FSMv1 flow that all these keys are strings. I haven't yet tested FSMv2. I believe GPO passes as an instance of `Pii::Attributes` which supports indifferent access.

```
[2] pry(main)> Pii::Attributes.new_from_hash(Idp::Constants::MOCK_IDV_APPLICANT_WITH_PHONE)['first_name']
=> "FAKEY"
```